WhaleSpotting/Services/SightingsService.cs

```diff
@@ -54,7 +55,7 @@ public async Task<List<SightingResponseModel>> SearchSighting(SearchSightingRequ
                 .Where(s => string.IsNullOrEmpty(searchSighting.Location) || s.Location == searchSighting.Location)
                 .Where(s => searchSighting.Confirmed == null || s.Confirmed == searchSighting.Confirmed)
                 .Include(s => s.User)
-                .OrderBy(s => s.SightedAt)
+                .OrderByDescending(s => s.SightedAt)
```

Comment:
Can we make everything orderby descending

Reply:
GetUserSightings
GetNotConfirmedSightings
GetAllSightings (Although they are on map, not ordered really)

Done